Makefile

```diff
@@ -9,11 +9,15 @@ format-check:
 	ls src/viam/ur/module/*.*pp | xargs clang-format-19 -i --style=file --dry-run --Werror
 
 configure:
-	cmake -S . -B build -G Ninja
+	cmake -S . -B build -G Ninja $(if $(DISABLE_APPIMAGE),-DVIAM_UR_DISABLE_APPIMAGE=ON)
```

Comment:
I've introduced a new Makefile flag called `DISABLE_APPIMAGE`, this just forwards it to the equivalent CMake option.